cmd/cloud/dbt.go

```diff
@@ -0,0 +1,150 @@
+package cloud
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cloud "github.com/astronomer/astro-cli/cloud/deploy"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	mountPath      string
+	dbtProjectPath string
+
+	DeployBundle = cloud.DeployBundle
+)
+
+const (
+	dbtDeployExample = `
+Specify the ID of the Deployment on Astronomer you would like to deploy this dbt project to:
+
+  $ astro dbt deploy -d <deployment ID>
```

Comment:
just to stay consistent with deploy flow where in we pass the deploymentId as part of the command and not via a flag.

`astro dbt deploy <deployment ID>`

Scanning through the code base, i see few instances where do use the flag. I think moving forward, we should pick one approach and stay consistent with it. IMO i like this flow better, since its something users are currently used to plus it leaves us the opportunity of using short hand flags for other purposes

Reply:
@neel-astro I think @kushalmalani is right, it is more consistent with the existing commands to have it as an argument. I'll change it back to that, but let me know if you strongly disagree.

Reply:
That `astro deploy` as a command doesn't refer to a resource is already confusing yeah. Thankfully there is no "dbt" resource in Astro that we would be getting mixed up with here.